pkg/cli/initconfig/cmd/init.go

```diff
@@ -60,13 +59,21 @@ var initCmd = &cobra.Command{
 	Short: "init system config",
 	Long:  `init system config.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		if err := run(); err != nil {
-			log.Fatal(err)
+		for {
```

Comment:
add a timeout internal, otherwise, the job won't stop if something is wrong. And don't forget to call `os.Exit(1)` if the job is failed